### ISSUE TITLE
Some code snipits in the 4.x document for the ray casting tutorial are still using the 3.x names.

### ISSUE THREAD

**Issue by JohnC3** [godotengine/godot-docs#7282]
From editor v4.0.2.stable.mono.official [7a0977ce2] 

The ray casting documentation at
https://docs.godotengine.org/en/stable/tutorials/physics/ray-casting.html for version 4 stable 

Includes the following code snipit

```
func _physics_process(delta):
    var space_rid = get_world_2d().space
    var space_state = Physics2DServer.space_get_direct_state(space_rid)
```

Physics2DServer appears to have been renamed to PhysicsServer2D. You might want to do a quick pass to see if any of this still works.

All the other places where they renamed stuff seems to have worked.

**Comment by JohnC3** [comment]
Don't know why I said all in my title I only found the one spot...

**Comment by Piralein** [comment]
Fixed by #7381.

(The fix will be visible in stable after it got cherry-picked)
